templates/user/settings/keys.tmpl

```diff
@@ -0,0 +1,144 @@
+{{template "base/head" .}}
```

Comment:
Could we break this file into 3? Having `settings/keys.tmpl`, `settings/keys_ssh.tmpl` and `settings/keys_gpg.tmpl`? Would make it easier to copy-paste a file to possibly add more key-types in the future. Also makes the files smaller ;)